fv3core/utils/grid.py

```diff
@@ -184,6 +189,32 @@ def jrange_domain(self):
     def krange(self):
         return range(0, self.npz)
 
+    # Added for convenience passing the GridIndexer to the
+    # fill_corners, while some objects still use grid
+    @property
+    def n_halo(self):
+        return self.halo
+
+    @property
+    def isc(self):
+        return self.is_
+
+    @property
+    def iec(self):
+        return self.ie
+
+    @property
+    def jsc(self):
+        return self.js
+
+    @property
+    def jec(self):
+        return self.je
+
+    @property
+    def domain(self):
+        return self.domain_shape_full()
```

Comment:
Can you tell me where you use these in this PR? It's a little hard to track it down.

Remember you can always pass `grid.grid_indexing` to a routine that expects a `GridIndexing` class.